authentication/service.js

```diff
@@ -35,11 +36,16 @@ module.exports = function () {
       create: [
         authenticate(config.strategies)
       ]
+    },
+    after: {
+      all: [
+        addCurrentAgent
```

Comment:
this is adding the hook to the authentication methods, we instead want to add it as a ["application hook"](https://docs.feathersjs.com/api/hooks.html#application-hooks), so it's present on _all_ methods across _all_ services. seems we do this with `app.hooks({ before: [ addCurrentAgent ] })`

Reply:
this would probably best go in https://github.com/root-systems/dogstack-agents/blob/master/service.js